Maxmin.js

```diff
@@ -35,4 +35,4 @@ function minmax(a, b, c) {
     largest(a, b, c);
 
 }
-minmax(3, 4, 6);
\ No newline at end of file
+minmax(5, 4, 6);
```

Comment:
- in largest: use curly braces in your if-else, do not write if-else statements without curly braces even if it works.